### PR TITLE
Add lark data and loadData.mjs script

### PR DESCRIPTION
**Pull request by toolgirl**
Fixed bug with scheme definitions in register endpoint

The form looks good, but the price is currently not being calculated. Investigating.

### CODE REVIEW

**Comment by willfulbard** [comment]
I think that this is fine to merge for now, since everything is a little in flux and we aren't deployed yet

File: package.json [hunk 1]
```diff
@@ -1,5 +1,9 @@
 {
   "scripts": {
     "heroku-postbuild": "cd client && yarn install && yarn run build"
+  },
+  "devDependencies": {
```
Comment (willfulbard) [inline package.json:5]: I don't exactly see where these are used at the moment...

File: server/camphoric/views.py [hunk 2]
```diff
@@ -155,9 +155,11 @@ def post(self, request, event_id=None, format=None):
     def get_form_schema(cls, event):
         if not event.registration_schema:
             return None
+        print(event)
```
Comment (brsaylor) [inline server/camphoric/views.py:158]: Need to remove this

File: package.json (hunk 1 above)
Comment (willfulbard) [reply package.json:5]: Also, these deps should probably live in the data directory

**Pull request merged**